gto/git_utils.py

```diff
@@ -67,3 +67,17 @@ def cloned_git_repo(repo: str):
 def git_clone(repo: str, dir: str) -> None:
     logging.debug("clone %s in directory %s", repo, dir)
     Repo.clone_from(url=repo, to_path=dir)
+
+
+def git_push_tag(repo_path: str, tag_name: str, remote_name: str = "origin") -> None:
+    repo = Repo(path=repo_path)
+    remote = repo.remote(name=remote_name)
+    remote_url = "unknown" if not hasattr(remote, "url") else remote.url
+    logging.debug(
```

Comment:
I think it would be good if we would be verbose here and communicate how the command changed the world outside of it :)

Like always printing this to stdout:
```shell
$ gto register model --auto-push
Created Git tag `model@v0.0.1`
Running `git push origin model@v0.0.1`
Successfully pushed Git tag(s) to the remote.
```

Or even just printing the real command output (I'm sure `GitPython` calls `git` CLI underneath, so maybe we can just get that output and print it). In this case it would behave just as if you run that command, which is well recognizeable and familiar.

Reply:
Yes, looks good!